<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotusOperationView</class>
 <widget class="QDialog" name="MotusOperationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MotusOperationView{ border-image: url(:/picture/playbackground.png);}
</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainGridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="topHLayout">
      <item>
       <layout class="QVBoxLayout" name="leftVLayout">
        <item>
         <layout class="QHBoxLayout" name="buttonHLayout">
          <item>
           <widget class="QPushButton" name="playFileLoad">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(70, 198,55);
border-radius:5px;
}
QPushButton:hover{color: rgb(255,255,255);} QPushButton:pressed{padding-left:3px;padding-top:3px;}</string>
            </property>
            <property name="text">
             <string>开机</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/picture/kaiji.png</normaloff>:/picture/kaiji.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playRestartButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(0, 227,234);
border-radius:5px;
}
QPushButton:hover{color: rgb(255,255,255);} QPushButton:pressed{padding-left:3px;padding-top:3px;}</string>
            </property>
            <property name="text">
             <string>复位</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/picture/reset.png</normaloff>:/picture/reset.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playStopButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(28, 161,218);
border-radius:5px;
}
QPushButton:hover{color: rgb(255,255,255);} QPushButton:pressed{padding-left:3px;padding-top:3px;}</string>
            </property>
            <property name="text">
             <string>运行</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/picture/run.png</normaloff>:/picture/run.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="statusHLayout">
          <item>
           <widget class="QLabel" name="playCurrentPlatfromStatus">
            <property name="minimumSize">
             <size>
              <width>145</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>当前平台状态：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="playCurrentFlowPath">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-width:0;border-style:outset;background:transparent;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="errorHLayout">
          <item>
           <widget class="QLabel" name="playCurrentPlatfromMessage">
            <property name="minimumSize">
             <size>
              <width>145</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>平台错误信息：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="playCurrentErrorMessage">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border-width:0;border-style:outset;background:transparent;</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>338</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="rightHLayout">
        <item>
         <widget class="QTableView" name="playTableView">
          <property name="minimumSize">
           <size>
            <width>270</width>
            <height>361</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="dataVLayout">
          <item>
           <widget class="QPushButton" name="selectFileButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(122, 175,207);
border-radius:5px;
}
QPushButton:hover{color: rgb(255,255,255);} QPushButton:pressed{padding-left:3px;padding-top:3px;}</string>
            </property>
            <property name="text">
             <string>选择文件</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/picture/openfile.png</normaloff>:/picture/openfile.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="movieLabel">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>电影名称</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="movieLineEdit">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-width:1;border-style:outset;background:transparent;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="datalabel">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>数据名称</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dataLineEdit">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-width:1;border-style:outset;background:transparent;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sequenceLabel">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>电影序列</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="sequenceLineEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-width:1;border-style:outset;background:transparent;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveFileButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(122, 175,207);
border-radius:5px;
}
QPushButton:hover{color: rgb(255,255,255);} QPushButton:pressed{padding-left:3px;padding-top:3px;}</string>
            </property>
            <property name="text">
             <string>保存文件</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/picture/savefile.png</normaloff>:/picture/savefile.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteFileButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>楷体</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(96, 187,214);
border-radius:5px;
}
QPushButton:hover{color: rgb(255,255,255);} QPushButton:pressed{padding-left:3px;padding-top:3px;}</string>
            </property>
            <property name="text">
             <string>删除文件</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/picture/deletefile.png</normaloff>:/picture/deletefile.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>45</width>
              <height>45</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="playDetialVLayout">
      <item>
       <layout class="QHBoxLayout" name="rateHLayout">
        <item>
         <widget class="QLabel" name="playPlayProgress">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>播放进度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="playProgressBar">
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>25</height>
           </size>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="detailsLayout">
        <item>
         <widget class="QLabel" name="playMovie">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>当前影片</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="playMovieEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-width:0;border-style:outset;background:transparent;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="playData">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>数据文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="playDataEdit">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-width:0;border-style:outset;background:transparent;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="playMovieSequence">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>电影序列</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="playMovieSequenceEdit">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-width:0;border-style:outset;background:transparent;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
